src/ios/OpenTokPlugin.m

```diff
@@ -7,6 +7,9 @@
 
 #import "OpenTokPlugin.h"
 
+int publisherZIndex = 1;
+int subscriberZIndex = 0;
```

Comment:
add comment in `initPublisher` and `subscribe` functions to explain that the given zIndex will never be used (also, maybe we should fix the zIndex given by the JS instead of removing logic here ?)